### PR TITLE
Test for pl <Testing

### PR DESCRIPTION
**Pull request by NilsHullegien**
Created more tests to increase code coverage.
<Testing

### CODE REVIEW

File: core/src/test/java/com/group5/core/WorldManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        worldManager.setPlayer(p1);
+    }
+
+    /**
+     * Check whether you can correctly set and get the current player
+     */
+    @Test
+    public void setPlayerTest() {
+        worldManager.setPlayer(p1);
+        assertTrue(worldManager.getPlayer().equals(new Player(physicsWorld, new Vector2(0, 0), new Vector2(20, 20))));
```
Comment (mwolting) [inline core/src/test/java/com/group5/core/WorldManagerTest.java]: Maybe use assertEquals here?

**Comment by mwolting** [comment]
Looking good, no major issues

File: core/src/test/java/com/group5/core/WorldManagerTest.java (hunk 1 above)
Comment (NilsHullegien) [inline core/src/test/java/com/group5/core/WorldManagerTest.java]: Good idea, will change.

**Comment by hmyeh** [comment]
Looks good

**Pull request merged**